ckanext/canada/model.py

```diff
@@ -0,0 +1,87 @@
+# encoding: utf-8
+
+import datetime
+import logging
+
+from sqlalchemy import Column, Unicode, DateTime, Integer
+from sqlalchemy.ext.declarative import declarative_base
+
+from ckan.model import meta
+
+log = logging.getLogger(__name__)
+
+
+Base = declarative_base(metadata=meta.metadata)
+
+
+class PackageSync(Base):
+    __tablename__ = 'package_sync'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    package_id = Column(Unicode)
```

Comment:
The package_id can be the PK, right?

Reply:
Yeee, just gunna keep it called `package_id` so we know what the reference is. Not gunna do any crazy backrefs for this table it that is fine?